x-pack/plugins/security_solution/public/cases/components/user_action_tree/index.tsx

```diff
@@ -40,6 +53,31 @@ const MyEuiFlexGroup = styled(EuiFlexGroup)`
   margin-bottom: 8px;
 `;
 
+const MyEuiCommentList = styled(EuiCommentList)`
+  ${({ theme }) => `
+    & .userAction__comment.outlined .euiCommentEvent {
+      outline: solid 5px ${theme.eui.euiColorVis1_behindText};
+      margin: 0.5em;
+      transition: 0.8s;
+    }
+
+    & .euiComment.isEdit {
+      & .euiCommentEvent {
+        border: none;
+        box-shadow: none;
+      }
+
+      & .euiCommentEvent__body {
+        padding: 0;
+      }
+
+      & .euiCommentEvent__header {
+        display: none;
+      }
+    }
+  `}
+`;
+
```

Comment:
its not letting me comment on the line, and i know you didnt touch this... but on L128 `handleSaveComment` deps need to be updated from `[caseData, handleManageMarkdownEditId, patchComment, updateCase]` to `[caseData.id, fetchUserActions, patchComment, updateCase]` in order to remove that `// eslint-disable-next-line react-hooks/exhaustive-deps`